lib/node_modules/@stdlib/number/float64/to-json/README.md

```diff
@@ -0,0 +1,99 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2022 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# number2json
+
+> Return a `JSON` representation of a `number`.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var number2json = require( '@stdlib/number/float64/to-json' );
+```
+
+#### number2json( x )
+
+Returns a `JSON` representation of a `number`.
```

Comment:
```suggestion
Returns a JSON representation of a number.
```

Our conventions have changed a bit (however, older conventions can still be found throughout the project), such as typically we only place in backticks variable or option names, not built-in data types.